src/pages/Ingame/index.js

```diff
@@ -95,16 +76,14 @@ export default function Game() {
         setTotalResult(tableData);
         setisFinishResult(true);
         setIsBoardModal(true);
-        // 최종 결과 보여주기
-        console.log("done", selectBoard, scoreBoard, currentTeamScore);
         return;
       }
       setIsSubmitted(false);
       setRoundDone(false);
       setMycard("");
-      round += 1;
-      checkSpecialRound();
-      history.push(`/game/:${roomName}-${team}-${round}`);
+      setRound(round + 1);
```

Comment:
혹시 이 부분이 아래처럼 바뀐다면 현재 라운드에서 1을 더해주니 바로 다음 라운드로 가는 구나라고 더 이해가 쉬워보일것 같습니다.
```suggestion
      setRound(currentRound + 1);
```